$OpenBSD: patch-src_tcp_listener_cpp,v 1.2 2014/09/23 12:08:03 ajacoutot Exp $

OpenBSD doesn't define EPROTO (yet), so pick the closest thing.

--- src/tcp_listener.cpp.orig	Wed Nov 13 18:57:50 2013
+++ src/tcp_listener.cpp	Mon Sep 22 14:06:18 2014
@@ -273,7 +273,7 @@ zmq::fd_t zmq::tcp_listener_t::accept ()
 #else
     if (sock == -1) {
         errno_assert (errno == EAGAIN || errno == EWOULDBLOCK ||
-            errno == EINTR || errno == ECONNABORTED || errno == EPROTO ||
+            errno == EINTR || errno == ECONNABORTED || errno == EOPNOTSUPP ||
             errno == ENOBUFS || errno == ENOMEM || errno == EMFILE ||
             errno == ENFILE);
         return retired_fd;
